coalib/coala_modes.py

```diff
@@ -61,8 +59,10 @@ def mode_json(args, debug=False):
 
     if args.show_bears:
         try:
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, LogPrinter())
+            from coalib.parsing.FilterHelper import FilterHelper
+
+            local_bears, global_bears = FilterHelper.get_filtered_bears(
+                'language', args.filter_by_language)
```

Comment:
I think we need a flag to get the list of languages apart from `filter_by_language` (`-l`). May be `--language`. Or we gotta change the meaning of `-l` to language, as filter_by_language doesn't always make sense.

This condition is not making sense to me (`if args.show_bears`). We are calling the function `mode_json` from coala if `args.json` is true. IMO, we should do processing using normally, and at the time of dumping data to stdout (using console_printer), we should check if the output format is chosen by user to some specific format(`json` in this case).
This seems like repeatation of code at the moment.